test/throttle_test.dart

```diff
@@ -64,8 +65,20 @@ void main() {
           values.add(2);
           await new Future(() {});
           await values.close();
+          await new Future(() {});
```

Comment:
This just wants to wait until the end of the current microtask queue.
You can use either `await null;` or `await new Future.value(null);` to the same effect, which should be cheaper than allocating a closure and calling it later.